src/core/Engine.cpp

```diff
@@ -22,6 +22,7 @@
 #include <log/logging.hpp>
 #include <original-content/OriginalGameFiles.hpp>
 #include <original-content/VirtualFileSystem.hpp>
+#include "original-content/OriginalGameResources.hpp"
```

Comment:
I would stick with one include style. I personally prefer `<>`. The includes are also not sorted alphabetically (might be because of `""` vs `<>`)

Reply:
Right, no idea how that slipped in!